front/app/components/IdeaCard/index.tsx

```diff
@@ -135,10 +135,10 @@ const IdeaCard = ({
 
   return (
     <Container
+      onClick={handleClick}
+      aria-hidden={true}
       className="e2e-card e2e-idea-card"
```

Comment:
If we set an onClick but make this container hidden, I think we should still maintain the behaviour we want for sighted users using a mouse/touchscreen?

Then we use the Link properly around the Title/Description for keyboard and screenreader users?